.gitlab-ci-check-golang-lint.yml

```diff
@@ -40,3 +40,18 @@ test:static:
     - fi
   script:
     - golangci-lint run -v
+
+test:govendor-check:
+  stage: test
+  needs: []
+  variables:
+    - GOLANG_VERSION=1.16
+  except:
+    - /^(master|hosted|staging|production|[0-9]+\.[0-9]+\.x|^saas-[a-zA-Z0-9.]+)$/
+  image: golang:${GOLANG_VERSION}
+  script:
+    - if [ -d vendor ]; then
+    -   go mod tidy
+    -   go mod vendor
+    -   git diff | wc -l && exit 1
```

Comment:
Okey. As an alternative, you could use [`git diff-index` instead](https://github.com/mendersoftware/mender-docs/blob/master/.gitlab-ci.yml#L77-L79).